src/vs/workbench/api/browser/mainThreadEditorTabs.ts

```diff
@@ -69,7 +72,8 @@ export class MainThreadEditorTabs {
 				tabs.push({
 					group: group.id,
 					name: editor.getTitle(Verbosity.SHORT) ?? '',
-					resource: editor.resource
+					resource: editor.resource,
+					isActive: group.isActive(editor)
```

Comment:
Unsure if this is "active per group" and hence many active editors or just one active editor. Something to test.